src/components/DpInput/DpInput.vue

```diff
@@ -129,6 +129,12 @@ export default {
       required: true
     },
 
+    inputClasses: {
```

Comment:
I ask myself if this is the right approach... it is so "universal". We could as well use a prop `isGrouped: 'left' | 'right' | false' which would default to false. DpInput would then add the appropriate classes by itself.

Reply:
I thought that for passing css classes, we wanted this universal approach, so we don't have to pass a prop for every class we want to add?

Reply:
After looking into it some more, I think you're right. But do we really need the `'left'` value? I'd prefer to keep the prop a Boolean for now, which is also what the name indicates.

Reply:
Since I removed the right border of the search input [here](https://github.com/demos-europe/demosplan-core/pull/4193/commits/c1cf03c9335f65591e3a368aed4d0e6e9154712e), we actually don't need the `rounded-r-none` class anymore, so I removed the prop + class in 9e010604da9117765641a1d8d4755db5692d20b1